crates/db-access/src/models.rs

```diff
@@ -39,3 +39,26 @@ pub struct ApiKey {
     pub key: String,
     pub name: Option<String>,
 }
+
+#[derive(Debug, sqlx::Type)]
+pub enum JobStatus {
+    Completed,
+    Pending,
+    Failed,
+}
+
+impl JobStatus {
+    pub fn as_str(&self) -> &'static str {
+        match self {
+            JobStatus::Completed => "Completed",
+            JobStatus::Pending => "Pending",
+            JobStatus::Failed => "Failed",
+        }
+    }
+}
+
+#[derive(sqlx::FromRow, Debug)]
+pub struct JobRequest {
+    pub job_id: String,
+    pub status: JobStatus,
+}
```

Comment:
Should also add migration script for this so its reproducible. There should be some example under `migrations/`. 

In the future might be a good idea to separate the dbs eventually so that the responsibility is clear and the server can manage user space db schemas whilst the indexer manage indexer schemas.

Reply:
It makes sense, I like this idea